src/Calculator.cpp

```diff
@@ -2,10 +2,29 @@
 /**
  *  @file   Calculator.cpp
  *  @date   Fri April 16 2021
- *  @brief  Funtions that support main.cpp
+ *  @brief  These are the functions that are used by main.cpp
  */
 ////////////////////////////////////////////////////////////////////////////
 
 #include "Calculator.h"
 
-// Add Function definitions here.
\ No newline at end of file
+namespace Calculator
+{
+        //*************/
```

Comment:
Looks like tabbing is off, there should be 4 spaces

Reply:
I'm a little confused? In between the namespace and the class creation, there should be 4 enter spaces?